OpenTerm/View/TerminalTextView.swift

```diff
@@ -49,18 +49,17 @@ class TerminalTextView: UITextView {
 	}
 
 	private func updateAppearanceFromSettings() {
-		let userDefaultsController = UserDefaultsController.shared
+		let userDefaults = UserDefaults.terminalDefaults
 
-		let terminalFontSize = userDefaultsController.terminalFontSize
-		self.font = UIFont(name: "Menlo", size: CGFloat(terminalFontSize))
+		let fontSize: Int = userDefaults[.terminalFontSize]
```

Comment:
Hmm, I get the feeling that this enum approach is less type safe. One could accidentally try to assign to a wrong type, which would trigger a runtime crash. It's a great example of what's possible with Swift enums, but I believe it's the wrong abstraction for user defaults.